packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/AddTokenModal.tsx

```diff
@@ -127,34 +126,26 @@ export const AddTokenModal = ({ onCancel }: AddTokenModalProps) => {
             onCancel={onCancel}
             heading={<Translation id="TR_ADD_TOKEN_TITLE" />}
             bottomBarComponents={
-                showDebugMode ? (
```

Comment:
Add token modal feature should be removed? You added it also for solana and cardano by this

Reply:
I thought that we agreed that for debug purposes we want to just hide it behind debug mode since it can be handy for us or e. g. QA, not to remove it completely no?

But it should stay just for ETH, will fix that.

Reply:
<img width="914" alt="Screenshot 2024-05-29 at 17 42 11" src="https://github.com/trezor/trezor-suite/assets/33235762/9aa0b733-b0bb-4e03-823e-b8d0deda7ec7">
🤷‍♂️

Reply:
https://satoshilabs.slack.com/archives/C03SRP8PSS2/p1711451432420569?thread_ts=1711449844.007949&cid=C03SRP8PSS2

Reply:
a93c40f60d1bba016f7653823770b932fef79dfd